### PR TITLE
bugfix: complete json tag for StreamServerReusePort and add some useful log for stop & remove pod operation

### PR DESCRIPTION
**Pull request by YaoZengzeng**
### Ⅰ. Describe what this PR did

1. complete the missing json tag for `StreamServerReusePort`

2. add some useful logs for stop & remove pod which may have multiple containers

3. failed to log `createErr`

### Ⅲ. Why don't you add test cases (unit test/integration test)? (你真的觉得不需要加测试吗？)

No

### CODE REVIEW

File: cri/v1alpha1/cri.go [hunk 1]
```diff
@@ -394,6 +397,7 @@ func (c *CriManager) RemovePodSandbox(ctx context.Context, r *runtime.RemovePodS
 		if err != nil {
 			return nil, fmt.Errorf("failed to remove container %q of sandbox %q: %v", container.ID, podSandboxID, err)
 		}
+		logrus.Infof("successfully remove container %q of sandbox %q", container.ID, podSandboxID)
```
Comment (Starnop) [inline cri/v1alpha1/cri.go]: Maybe "success to do something" will be better? ^_^

File: cri/v1alpha1/cri.go [hunk 2]
```diff
@@ -329,6 +329,7 @@ func (c *CriManager) StopPodSandbox(ctx context.Context, r *runtime.StopPodSandb
 		if err != nil {
 			return nil, fmt.Errorf("failed to stop container %q of sandbox %q: %v", container.ID, podSandboxID, err)
 		}
+		logrus.Infof("success to stop container %q of sandbox %q", container.ID, podSandboxID)
```
Comment (allencloud) [inline cri/v1alpha1/cri.go]: Maybe `debug` is sufficient.

File: cri/v1alpha1/cri.go [hunk 3]
```diff
@@ -394,6 +397,7 @@ func (c *CriManager) RemovePodSandbox(ctx context.Context, r *runtime.RemovePodS
 		if err != nil {
 			return nil, fmt.Errorf("failed to remove container %q of sandbox %q: %v", container.ID, podSandboxID, err)
 		}
+		logrus.Infof("success remove container %q of sandbox %q", container.ID, podSandboxID)
```
Comment (allencloud) [inline cri/v1alpha1/cri.go]: Same issue. Do you think so?

**Comment by rudyfly** [comment]
LGTM, waiting for ci passed.

File: cri/v1alpha1/cri.go (hunk 3 above)
Comment (YaoZengzeng) [reply cri/v1alpha1/cri.go]: Hmm... I think maybe `Infof` is more suitable

The operation of `stop` and `remove` is not as frequent as list

It's helpful and important for us to know details of every pod related operation 😂

Comment (allencloud) [reply cri/v1alpha1/cri.go]: OK， just double check.

**Pull request merged**